tests/devices/grabber_audiovisual.ini

```diff
@@ -1,5 +1,5 @@
 # wrap an audiovisual source for the network
-device grabber
+device grabberDual
```

Comment:
This is not correct.
The test is tryong to broadcast both images and sound. The old `grabber` is still the only device capable of doing both at the same time. It is the only reason why the `grabber` device is not removed yet.

Reply:
Reverted this change